app/assets/javascripts/basetypes/DataElement.js

```diff
@@ -1,36 +1,74 @@
 const mongoose = require('mongoose');
 const Code = require('./Code.js');
+const changeCase = require('change-case');
 
 const [Schema] = [mongoose.Schema];
 
-function extendSchema(TSchema, definition, options) {
-  return new Schema(Object.assign({}, TSchema.obj, definition), options);
-}
+function DataElementSchema(add, options) {
+  const extended = new Schema({
+    data_element_codes: { type: [Code] },
+    description: { type: String },
+  }, options);
+
+  if (add) {
+    extended.add(add);
+  }
+
+  // TODO: REMOVE THIS. No longer needed after the change to camelCase
```

Comment:
Is something preventing you from removing this now?

Reply:
Yes. Until another PR is in that changes every patient and data element field to camel case, this conversion needs to happen.